music_assistant/server/providers/filesystem_smb/__init__.py

```diff
@@ -171,13 +171,13 @@ async def get_audio_stream(
 
     async def mount(self) -> None:
         """Mount the SMB location to a temporary folder."""
-        server: str = self.config.get_value(CONF_HOST)
-        username: str = self.config.get_value(CONF_USERNAME)
-        password: str = self.config.get_value(CONF_PASSWORD)
-        share: str = self.config.get_value(CONF_SHARE)
+        server = str(self.config.get_value(CONF_HOST))
+        username = str(self.config.get_value(CONF_USERNAME))
+        password = str(self.config.get_value(CONF_PASSWORD))
```

Comment:
this will turn None into a string "None"
I'm not sure if we allow empty passwords though

Reply:
Empty passwords seem to be allowed. Have updated.